commercetools/commercetools-sdk-java-api/src/main/java/com/commercetools/api/json/AttributeDeserializer.java

```diff
@@ -21,7 +21,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.JsonNodeType;
 
-public class AtrributeDeserializer extends JsonDeserializer<AttributeImpl> {
+public class AttributeDeserializer extends JsonDeserializer<AttributeImpl> {
```

Comment:
could you please add a class

```
@Deprecated
class AtrributeDeserializer extends AttributeDeserializer {}
```

so that it doesn't break anything if someone uses it with the wrong spelling. You may have to also add the constructors too.

Reply:
https://github.com/commercetools/commercetools-sdk-java-v2/pull/748/commits/7edd50abfd3c4878db0e2a9fb90d4e02c482edc5